infra/ansible/roles/fetch_srcs/tasks/main.yaml

```diff
@@ -20,25 +20,6 @@
       dest: "{{ (src_root, 'pytorch/xla') | path_join }}"
       version: "{{ xla_git_rev }}"
 
-- name: Find *.diff files in pytorch/xla/openxla_patches
```

Comment:
I wonder why we don't need the openxla patches anymore.

Reply:
The patches are applied by bazel automatically now.